python/ipywidgets/ipywidgets/widgets/tests/utils.py

```diff
@@ -16,16 +19,30 @@ def __init__(self, *args, **kwargs):
     def open(self, *args, **kwargs):
         pass
 
+    def on_msg(self, *args, **kwargs):
+        pass
+
     def send(self, *args, **kwargs):
         self.messages.append((args, kwargs))
 
     def close(self, *args, **kwargs):
         pass
 
+
+def dummy_create_comm(**kwargs):
+    return DummyComm()
+
+
+def dummy_get_comm_manager(**kwargs):
+    return {}
+
+
 _widget_attrs = {}
 undefined = object()
 
 def setup_test_comm():
+    comm.create_comm = dummy_create_comm
+    comm.get_comm_manager = dummy_get_comm_manager
```

Comment:
should we restore this? in a try/finally?

Reply:
We should 👍🏽 I will restore them in the teardown function